src/Components/v2/SelectedCareerAchievements.tsx

```diff
@@ -68,12 +68,13 @@ export class SelectedCareerAchievements extends React.Component<
     ) {
       const highCategory = highestCategory(partnersConnection.edges)
       const type = highCategory.toUpperCase().replace("-", "_")
+      const label = CATEGORIES[highCategory] + " representation"
```

Comment:
Same question here as above-- is `CATEGORIES` used anywhere else? If not can you just change the values to include `"representation"`?